flake.nix

```diff
@@ -6,11 +6,12 @@
     haskell-flake.url = "github:srid/haskell-flake";
     treefmt-flake.url = "github:srid/treefmt-flake";
     check-flake.url = "github:srid/check-flake";
+    deploy-rs.url = "github:serokell/deploy-rs";
   };
 
-  outputs = inputs@{ self, nixpkgs, flake-parts, ... }:
+  outputs = inputs@{ self, nixpkgs, flake-parts, deploy-rs, ... }:
     flake-parts.lib.mkFlake { inherit self; } {
-      systems = nixpkgs.lib.systems.flakeExposed;
+      systems = [ "x86_64-linux" ];
```

Comment:
Do you understand how this change (to `systems`) will impact macOS users like myself?

Reply:
I can work on this, we were getting an error with Darwin when we had it as it was previously.

Reply:
Probably something to do with deploy-rs checks. You can simply not use them, and use `deploy -s` to deploy.

Reply:
On second-look, whilst I was getting a build error with Darwin earlier.

```
error: a 'aarch64-darwin' with features {} is required to build '/nix/store/4l9v2iq6ynglvdrhpxh7mjprx6dyjyay-cabal2nix-mood-tracker.drv', but I am a 'x86_64-linux' with features {benchmark, big-parallel, nixos-test}
(use '--show-trace' to show detailed location information)
```

Upon reverting this to how it was the problem doesn't appear to manifest anymore.